pytest/lib/account.py

```diff
@@ -0,0 +1,75 @@
+import base64
+import json
+import requests
+import time
+
+from cluster import Key
+from transaction import (
+    sign_payment_tx, sign_deploy_contract_tx, sign_function_call_tx,
+    sign_create_account_with_full_access_key_and_balance_tx, sign_staking_tx)
+import utils
+
+
+class Account:
+
+    def __init__(self, key, init_nonce, base_block_hash, rpc_info):
+        self.key = key
+        self.nonce = init_nonce
+        self.base_block_hash = base_block_hash
+        self.rpc_addr, self.rpc_port = rpc_info
+        self.tx_timestamps = []
```

Comment:
why do we need this?

Reply:
It allows much more detailed measurements of sending transactions to the nodes. For example, one of the lines on the graphs I showed a couple weeks ago (with the spiked input tps) were generated from capturing these timestamps.